OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -367,16 +398,18 @@ protected void BeginProduction(ProductionItem item)
 		// Returns the actor/trait that is most likely (but not necessarily guaranteed) to produce something in this queue
 		public virtual TraitPair<Production> MostLikelyProducer()
 		{
-			var trait = self.TraitsImplementing<Production>().FirstOrDefault(p => p.Info.Produces.Contains(Info.Type));
+			var trait = productionTraits.FirstOrDefault(p => !p.IsTraitDisabled && !p.IsTraitPaused && p.Info.Produces.Contains(Info.Type));
```

Comment:
Similarly here: check if the first unpaused one is null and fall back to the first paused one.